### PR TITLE
Added more simplified example to OOPS

### PR DESCRIPTION
**Pull request by aslezar**
## Proposed changes

Added more simplified example of BookStore to OOPS section for engineer to understand it easily.

## Types of changes

What types of changes does your content introduce to this project?
_Put an `x` in the boxes that apply (without "`")_

- [ ] Proofreading (fix spelling, grammar, punctuation, accuracy)
- [x] New content (new section, new readme, etc.)
- [ ] Modifications (modifying existing content)
- [ ] Other (please describe)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. Don't hesitate to ask if you're unsure about any of them. We're here to help! This simply reminds us of what we will look for before merging your code._
_Note: If you submit code make sure to test it_

- [x] Tested any code that was added/modified in this PR
- [ ] Added external links
- [ ] Added internal links
- [ ] Added images
- [ ] Added videos
- [x] Added code snippets

### Reviewers: 
- [ ] @username

_Please ensure that you are not adding any sensitive information like API keys, secrets, passwords, tokens, usernames, or other credentials. Refrain from adding your name, email, or other personal information._

### CODE REVIEW

File: .all-contributorsrc
Comment (shhossain) [inline .all-contributorsrc:1]: Don't edit this file.

**Review by shhossain — CHANGES_REQUESTED** [review]

File: .all-contributorsrc
Comment (aslezar) [reply .all-contributorsrc:1]: OK, I have reverted that commit.

**Review by shhossain — APPROVED** [review]
No problem.

**Comment by shhossain** [comment]
@all-contributors add @aslezar for code.

**Pull request merged**